src/patternfly/components/ModalBox/styles.scss

```diff
@@ -33,31 +46,100 @@
   --pf-c-modal-box__footer--PaddingBottom: var(--pf-global--spacer--xl);
   --pf-c-modal-box__footer--PaddingLeft: var(--pf-global--spacer--xl);
 
+  // Box Footer Button variables
+  --pf-c-modal-box__footer--ButtonSpacer: var(--pf-global--spacer--md);
+  --pf-c-modal-box__footer--ItemSpacer: var(--pf-global--spacer--xs);
+
+  // Box button-close variables
+  --pf-c-modal-box__close--MarginRight: calc(var(--pf-global--spacer--sm) * -1);
+  --pf-c-modal-box__close--MarginTop: calc(var(--pf-global--spacer--sm) * -1);
+
   // This component always needs to be light
   @extend %pf-t-light;
 
   z-index: var(--pf-c-modal-box--ZIndex);
   display: flex;
   flex-direction: column;
-  max-width: var(--pf-c-modal-box--MaxWidth);
+  // Ensure modal has margins at breakpoint - sm
+  max-width: calc(100% - var(--pf-c-modal-box--MarginLeft) - var(--pf-c-modal-box--MarginRight));
   min-height: var(--pf-c-modal-box--MinHeight);
   max-height: var(--pf-c-modal-box--MaxHeight);
   background-color: var(--pf-c-modal-box--BackgroundColor);
   border: var(--pf-c-modal-box--BorderSize) solid var(--pf-c-modal-box--BorderColor);
   box-shadow: var(--pf-c-modal-box--BoxShadow);
 
+  // At breakpoint--sm, set max width to variable value and ignore margins
+  &:not(.pf-m-lg) {
+    @media screen and (min-width: $pf-global--breakpoint--sm) {
+      max-width: var(--pf-c-modal-box--MaxWidth);
+    }
+  }
+
+  // At breakpoint--xl, set max width to variable value and ignore margins
   &.pf-m-lg {
-    max-width: var(--pf-c-modal-box--MaxWidth--lg);
+    @media screen and (min-width: $pf-global--breakpoint--xl) {
+      max-width: var(--pf-c-modal-box--lg--MaxWidth);
+    }
   }
 
+  // Header
   &__header {
     flex: 0 0 auto;
     padding-top: var(--pf-c-modal-box__header--PaddingTop);
```

Comment:
According the docs I believe any instance where you are explicitly stating all the values you could use `padding:` This was already here but think we should update.